catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/BoundingBoxReader.java

```diff
@@ -166,4 +175,19 @@ private String createWkt(String[] lowerCornerPosition, String[] upperCornerPosit
         }
         }
     }
+
+    private String convertToEPSG4326(String wkt, String crs) {
+
+        try {
+            Geometry geometry = wktReader.read(wkt);
+            Geometry convertedGeometry = GeospatialUtil.transformToEPSG4326LonLatFormat(geometry, crs);
+            if (convertedGeometry != null) {
+                return wktWriter.write(convertedGeometry);
+            }
+        } catch (ParseException | GeoFormatException e) {
+            LOGGER.debug("Unable to convert {} from {} to EPSG:4326", wkt, crs, e);
```

Comment:
As it stands now, geometries that come in from a CSW Source that are not EPSG:4326 ( or the lat / lon is not properly ordered), there will be exceptions further down in the stack.  It seems like the best thing to do is to not set the geometry if the WKT cannot be parsed.